apps/03TicTacToe.js

```diff
@@ -24,23 +24,81 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+
+//  Your code here
+  if ((board[0][0] === 'X' && board[0][1] === 'X' && board[0][2] === 'X' ) || (board[0][0] === 'O' && board[0][1] === 'O' && board[0][2] === 'O')) {
```

Comment:
yikes.  I think i accidentally clipped it off when I copied & pasted to Diagonal & Vertical.

I will add it back.